win32ss/user/ntuser/message.c

```diff
@@ -12,9 +12,33 @@
 DBG_DEFAULT_CHANNEL(UserMsg);
 
 #define PM_BADMSGFLAGS ~((QS_RAWINPUT << 16)|PM_QS_SENDMESSAGE|PM_QS_PAINT|PM_QS_POSTMESSAGE|PM_QS_INPUT|PM_NOYIELD|PM_REMOVE)
+#define EXCEPTION_ACCESS_VIOLATION ((DWORD)0xC0000005)
+
+/* Strings that are OK to pass between user and kernel mode
+ * There may be other strings needed that can easily be added here. */
+WCHAR StrUserKernel[3][20] = {{L"intl"}, {L"Environment"}, {L"Policy"}};
```

Comment:
Why putting these strings inside "sublists"? Why not just doing something like:
```suggestion
PCWSTR StrUserKernel[] = {L"intl", L"Environment", L"Policy"};
```
?

Reply:
@HBelusca, I know that what I have here will allow me to use understandable derived values as follows:
ARRAYSIZE(StrUserKernel) = 3
sizeof(StrUserKernel[0]) = 40
_countof(StrUserKernel[0]) = 20
To me what you are suggesting seems to be the very definition of "ill-defined".
What would be the values above when the array is defined without integer constant numbers?
With what you have proposed, I cannot tell what the above values would return by simple inspection.
but with the values ```[3][20]``` it seems obvious to me. Admittedly, this is because of my poor understanding.
Nevertheless, I do appreciate your comments and your trying to help me with this. Thanks.